assets/js/components/surveys/CurrentSurvey.test.js

```diff
@@ -447,31 +447,58 @@ describe( 'CurrentSurvey', () => {
 			);
 		} );
 
-		it( 'should disable the submit button when the number of options selected is more than `maxChoices`', async () => {
-			const { getByText, getByRole } = render( <CurrentSurvey />, {
+		it( 'should disable other options once the number of options selected equals `maxChoices`', async () => {
+			const { getByLabelText, getByRole } = render( <CurrentSurvey />, {
 				registry,
 			} );
 
 			// Five items selected is too high and shoud cause the sub, button to be
 			// disabled.
-			fireEvent.click( getByText( 'Pepperoni' ) );
-			fireEvent.click( getByText( 'Sausage' ) );
-			fireEvent.click( getByText( 'Mushrooms' ) );
-			fireEvent.click( getByText( 'Black Olives' ) );
-			fireEvent.click( getByText( 'Sweetcorn' ) );
+			fireEvent.click( getByLabelText( 'Pepperoni' ) );
+			fireEvent.click( getByLabelText( 'Sausage' ) );
+			fireEvent.click( getByLabelText( 'Mushrooms' ) );
 
-			expect( getByRole( 'button', { name: 'Submit' } ) ).toHaveAttribute(
+			// This option will be enabled because we still haven't selected the
+			// maximum number of items.
+			expect( getByLabelText( 'Sweetcorn' ) ).not.toHaveAttribute(
```

Comment:
Shouldn't we be using [`(not.)toBeDisabled`](https://github.com/testing-library/jest-dom#tobedisabled) rather than checking the presence of the attribute? The matcher (which we have available via `jest-dom`) is quite a bit more robust than this so I would prefer we use that unless we have a specific reason why we can't.